examples/covid/Covid Using Graphs in Jupyter Notebook.ipynb

```diff
@@ -0,0 +1,487 @@
+    "%import_dataset covid19.alberta.case-details --alias cases_ab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7b1540af",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "Time to test again if the data was imported correctly. You can test the Alberta files by changing  <code>cases_on</code> to   <code>cases_ab</code>. \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5c50f473",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%%sql\n",
+    "SELECT * FROM cases_on\n",
+    "ORDER BY id DESC\n",
+    "LIMIT 10"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de367230",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "Now we are importing a yaml file which allows for changing the age ranges of each of the three provinces so that they align  with eachother. For example only one province has a 90+ whereas the other two has age ranges of 80+. Therefore we need to switch the age ranges to 80+ to compare the data.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e3c64739",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "%import_dataset abbcon.case-details --alias cases_three"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cb5fe9f",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "Time to test again if the data was imported correctly.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "083d28ed",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%%sql\n",
+    "SELECT * FROM cases_three\n",
+    "LIMIT 10"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d33ec36",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">With the below SQL querry you can compare the cases per age range per province\n",
+    " </div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e57b2944",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sql\n",
+    "SELECT province, age_group, COUNT(*) \n",
+    "FROM cases_three\n",
+    "GROUP BY province, age_group\n",
+    "ORDER BY province, age_group;"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c2a7e36",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "Since Quebec does not show data by age, we are importing this data now. However, if you want to test whether the data is correct, you would repeat the SQL query used for case_bc and case_on, but change id to row_id since Quebec uses different headers. This causes issues when combining data which will be talked about more later.\n",
```

Comment:
it looks like when I was looking through the quebec dataset I saw a lot of 'Not Reported', but now going through the full dataset, I realize that there definately is enough data so I can add the age_group to the other data.

Reply:
I heavily modified jupyter notebook so that Quebec age group data is shown and described how do to privacy laws why there is so much unknow age for the Quebec data